openmc/universe.py

```diff
@@ -7,7 +7,8 @@
 import numpy as np
 
 import openmc
-from openmc.checkvalue import check_type, check_length, check_greater_than
+from openmc.checkvalue import check_type, check_iterable_type, check_length, \
```

Comment:
At the point where we begin to import every routine from a submodule, is it worth just calling ``import openmc.checkvalue as checkvalue`` and then calling routines as ``checkvalue.check_type(...)``, etc.? Or even doing the deepest, darkest thing possible and calling ``from openmc.checkvalue import *``?